<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
<test-results name="C:\Users\20211148060017\Desktop\workspace\testes-corretor-delphi\Win32\Debug\Project1.exe" total="9" errors="0" failures="2" ignored="0" inconclusive="0" not-run="0" skipped="0" invalid="0" date="2023-04-26" time="0.194">
  <culture-info current-culture="en" current-uiculture="en" />
  <test-suite type="Assembly" name="Project1.exe" executed="true" result="Failure" success="False" time="0.194" asserts="0">
    <results>
        <test-suite type="Namespace" name="Unit2" executed="true" result="Failure" success="False" time="0.193" asserts="0" >
          <results>
              <test-suite type="Fixture" name="TMyTestObject" executed="True" result="Failure" success="False" time="0.193" >
                <results>
                  <test-case name="TestComissao.Comissao 01" executed="True" result="Success" success="True" time="0.000" asserts="0" />
                  <test-case name="TestComissao.Comissao 02" executed="True" result="Failure" success="False" time="0.000" asserts="0" >
                    <failure>
                      <message>
                        <![CDATA[ Expected 7200 but got 600001  ]]>
                      </message>
                      <stack-trace>
                        <![CDATA[  ]]>
                      </stack-trace>
                    </failure>
                  </test-case>
                  <test-case name="TestNome.Nome" executed="True" result="Success" success="True" time="0.000" asserts="0" />
                  <test-case name="TestData.Data" executed="True" result="Failure" success="False" time="0.000" asserts="0" >
                    <failure>
                      <message>
                        <![CDATA[ Method did not throw any exceptions. ]]>
                      </message>
                      <stack-trace>
                        <![CDATA[  ]]>
                      </stack-trace>
                    </failure>
                  </test-case>
                  <test-case name="TestSalario.Salario" executed="True" result="Success" success="True" time="0.000" asserts="0" />
                  <test-case name="TestValorVendidoInvalido.testar valor vendido inválido" executed="True" result="Success" success="True" time="0.000" asserts="0" />
                  <test-case name="TestValorVendidoInvalido.testar valor vendido inválido" executed="True" result="Success" success="True" time="0.000" asserts="0" />
                  <test-case name="TestValorVendidoValido.testar valor vendido inválido" executed="True" result="Success" success="True" time="0.000" asserts="0" />
                  <test-case name="TestValorVendidoValido.testar valor vendido inválido" executed="True" result="Success" success="True" time="0.000" asserts="0" />
                </results>
              </test-suite>
          </results>
        </test-suite>
    </results>
  </test-suite>
</test-results>
